src/main/java/com/curler/os/domains/OS.java

```diff
@@ -0,0 +1,172 @@
+package com.curler.os.domains;
+
+import com.curler.os.domains.enuns.Prioridade;
+import com.curler.os.domains.enuns.Status;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDateTime;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+
+@Entity
+public class OS {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    
+    @JsonFormat(pattern = "dd/MM/yyyy HH:mm")
+    private LocalDateTime dataAbertura;
+    
+    @JsonFormat(pattern = "dd/MM/yyyy HH:mm")
+    private LocalDateTime dataFechamento;
+    
+    private Integer prioridade;
+    
+    private String observacoes;
+    
+    private Integer status;
+    
+    private Tecnico tecnico;
+    
+    private Cliente cliente;
+
+    public OS() {
+        super();
+        
+        this.setDataAbertura(LocalDateTime.now());
+        this.setPrioridade(Prioridade.BAIXA);
+        this.setStatus(Status.ABERTO);
+    }
+
+    public OS(Integer id, Prioridade prioridade, String observacoes, Status status, Tecnico tecnico, Cliente cliente) {
+        this.id = id;
+        this.setDataAbertura(LocalDateTime.now());
+        this.prioridade = (prioridade == null)? 0 : prioridade.getCode();
+        this.observacoes = observacoes;
+        this.status = (status == null)? 0 : status.getCode();
+        this.tecnico = tecnico;
+        this.cliente = cliente;
+    }
+    
+    
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public LocalDateTime getDataAbertura() {
+        return dataAbertura;
+    }
+
+    public void setDataAbertura(LocalDateTime dataAbertura) {
+        this.dataAbertura = dataAbertura;
+    }
+
+    public LocalDateTime getDataFechamento() {
+        return dataFechamento;
+    }
+
+    public void setDataFechamento(LocalDateTime dataFechamento) {
+        this.dataFechamento = dataFechamento;
+    }
+
+    public Integer getPrioridade() {
+        return prioridade;
+    }
```

Comment:
public Prioridade getPrioridade() {
        return Prioridade.toEnum(this.prioridade);
 }

Reply:
Aqui pediu para mudar o tipo de retorno no método.